contracts/protocore/Protocore.sol

```diff
@@ -96,6 +96,9 @@ contract Protocore is MosaicVersion, CoconsensusModule, ForwardValidatorSetAbstr
     /** Epoch length */
     uint256 public epochLength;
 
+    /** Vote message hash of the latest finalized link */
+    bytes32 public latestFinalizedVoteMessageHash;
```

Comment:
This storage cost is going to be doubled as we have this information already in Coconsensus.

Reply:
Removed.